Source/JavaScriptCore/runtime/StructureID.h

```diff
@@ -34,12 +34,28 @@ namespace JSC {
 
 class Structure;
 
```

Comment:
I see that you did not move structureHeapAddressSize here from JSCConfig.h too.  Maybe do that in a follow up patch?  I don't see a reason it should be define there.

Reply:
Ah, yeah, for now, I'm keeping it since it exists before this patch.